tools/switch.sh

```diff
@@ -260,6 +262,26 @@ if [ $# -lt 1 ]; then
     exit 1
 fi
 
+function ep-tunnel-command() {
+    local SWITCH=$1 CMD=$2
+    echo "Execute switch ep-tunnel $CMD on $SWITCH"
+
+    DRIVES=$(switchtec fabric gfms-dump "$SWITCH" | grep "SRIOV-PF" -A2 | grep PDFID | awk '{print $2}')
+    for DRIVE in $DRIVES;
+    do
+        case $CMD in
```

Comment:
I'm not sure I would add this complexity. Why not just always echo the drive? Why only for "status"?

Reply:
There is no output for the ep-tunnel 'enable' command, so it looks pretty ugly.

Reply:
Oh so the printf outputs get smushed on one line. OK. Approved.